src/services/dashboards/widgets/_components/WidgetFrame.vue

```diff
@@ -6,74 +6,82 @@
         <div class="widget-header">
             <h3 class="title">
                 {{ title }}
-            </h3> <slot name="header-right" />
+            </h3><slot name="header-right" />
         </div>
         <div class="body">
             <slot />
         </div>
         <div class="widget-footer">
             <div class="footer-left">
-                <p-datetime-picker style-type="text"
-                                   select-mode="range"
-                                   :selected-dates.sync="proxySelectedDates"
-                />
-                <p-divider :vertical="true" />
-                <currency-select-dropdown :print-mode="printMode"
-                                          @update="handleUpdateCurrency"
+                <label v-if="dateLabel"
+                       class="widget-footer-label"
+                >{{ dateLabel }}</label>
+                <p-divider v-if="isDivided"
+                           :vertical="true"
                 />
+                <label class="widget-footer-label">{{ currencyLabel }}</label>
             </div>
             <div class="footer-right">
                 <slot name="footer-right">
-                    <router-link v-if="widgetLink && !printMode"
-                                 :to="widgetLink"
-                                 class="anchor-button"
+                    <component :is="fullDataTag"
+                               v-if="(widgetLink || widgetRoute) && !printMode"
```

Comment:
nitpick (non-blocking): state name for dynamic component & reduce code by removing `v-if` sentence

1. The name 'fullDataTag' sounds it can take only element tags such as 'span' or 'a'.
But <router-link> component also can be assigned to this state.
What about using more clear word such as 'fullDataLinkComponent'?

3. If the dynamic component is `null`, it renders nothing.
That means, we can move `v-if` sentence into fullDataTag computed hook.

Reply:
suggestion (blocking): Suggest to use `<p-anchor>` component.

In figma, designers used anchor component for 'full data' link.
I think, we should use `<p-anchor>` component to avoid fragmentation.

Reply:
You can ignore these review.